### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by mayuresh-tech**
Work done:
Color scheme
Sidebar bug fix
Documentation added
Links added

### CODE REVIEW

**Comment by BALAJI141031** [comment]
suggestioj:you can implement rating functionality

File: css/extra--large-media-query.css [hunk 1]
```diff
@@ -171,7 +185,11 @@ img {
 }
 
 iframe {
-  border: var(--primary-light-font-color) 1px solid;
+  border: var(--primary-light-color) 1px solid;
   width: 100%;
   min-height: 300px;
```
Comment (manish-GH) [inline css/extra--large-media-query.css:190]: use rem

File: css/large-media-query.css [hunk 2]
```diff
@@ -226,7 +233,11 @@ button#burger {
 }
 
 iframe {
-  border: var(--primary-light-font-color) 1px solid;
+  border: var(--primary-light-color) 1px solid;
   width: 100%;
   min-height: 300px;
```
Comment (manish-GH) [inline css/large-media-query.css:238]: use rem

File: css/medium-media-query.css [hunk 3]
```diff
@@ -226,7 +233,11 @@ button#burger {
 }
 
 iframe {
-  border: var(--primary-light-font-color) 1px solid;
+  border: var(--primary-light-color) 1px solid;
   width: 100%;
   min-height: 300px;
```
Comment (manish-GH) [inline css/medium-media-query.css:238]: use rem

File: css/small-media-query.css [hunk 4]
```diff
@@ -227,7 +233,11 @@ button#burger {
 }
 
 iframe {
-  border: var(--primary-light-font-color) 1px solid;
+  border: var(--primary-light-color) 1px solid;
   width: 100%;
   min-height: 300px;
```
Comment (manish-GH) [inline css/small-media-query.css:238]: use rem

**Review by manish-GH — APPROVED** [review]
Everything else looks good, can be merged

File: css/extra--large-media-query.css [hunk 5]
```diff
@@ -52,16 +49,22 @@ img {
   flex-grow: 1;
 }
 
-#nav-section-mobile, #mobile-section {
+#nav-section-mobile,
+#mobile-section {
   display: none;
 }
 
 #logo {
```
Comment (deekshithmd) [inline css/extra--large-media-query.css:57]: try to use class instead of id

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**